cloudflare/cloudflare-docs
# Rules --> Add top-level Examples gallery

### Summary

- [x] [Stream] Massage and metadata
- [x] Other products --> Pull into alignment to keep current behavior
- [x] Rules --> Pull everything together at the top level

### How to review

- Check out the main [Rules examples](https://examples-tweaks.cloudflare-docs-7ou.pages.dev/rules/examples/) page.
- Check out sub-pages w/in the Rules to make sure only those [examples are showing](https://examples-tweaks.cloudflare-docs-7ou.pages.dev/rules/snippets/examples/)
- Review individual examples page to check that the behavior is better or the same as previous (Stream / Vectorize / Queues / D1 / etc).
